src/stories/CardSimple.stories.js

```diff
@@ -0,0 +1,83 @@
+import UnnnicSimpleCard from '../components/Card/SimpleCard.vue';
+import UnnnicButton from '../components/Button/Button.vue';
+
+export default {
+  title: 'Data Display/SimpleCard',
+  component: UnnnicSimpleCard,
+  tags: ['autodocs'],
+  argTypes: {
+    title: { control: { type: 'text' } },
+    titleTooltip: { control: { type: 'text' } },
+    text: { control: { type: 'text' } },
+    titleSize: { control: { type: 'text' } },
+    clickable: { control: { type: 'boolean' } },
+  },
+  render: (args) => ({
+    components: {
+      UnnnicSimpleCard,
+    },
+    setup() {
+      return { args };
+    },
+    template: `
+        <UnnnicSimpleCard v-bind="args">
+        </UnnnicSimpleCard>
+    `,
+  }),
+};
+
+const DefaultTemplate = (args) => ({
+  components: { UnnnicSimpleCard },
+  setup() {
+    return { args };
+  },
+  template: `
+        <UnnnicSimpleCard v-bind="args">
+        </UnnnicSimpleCard>
+    `,
+});
```

Comment:
I believe the same rule for automatic component rendering works here, you can change it to this:

```suggestion
export const Default = {
  args: {
     title: 'This is the title',
    titleTooltip: 'Tooltip',
    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis volutpat enim quis nisi laoreet eleifend. Nulla facilisi. Sed porttitor ut felis ut aliquam. Praesent interdum suscipit malesuada. Morbi rutrum, nunc ut aliquam fringilla, metus augue ultricies dui, vitae cursus risus purus ac nulla. Mauris dignissim bibendum eros sed congue. Praesent nec elit nec justo rhoncus tempus ut sit amet leo. Donec eu efficitur dolor.',
  },
};
```